instances/latest/commonCoordinateSpaceVersions/MNI-Colin27/MNI-Colin27_2008.jsonld

```diff
@@ -1,43 +1,54 @@
-	"isAlternativeVersionOf": [
-		{
-			"@id": "https://openminds.om-i.org/instances/commonCoordinateSpaceVersion/MNIColin27_1998"
-		}
-	],
-	"isNewVersionOf": null,
-	"keyword": null,
-	"license": null,
-	"nativeUnit": {
-			"@id": "https://openminds.om-i.org/instances/unitOfMeasurement/millimeter"
-	},
-	"ontologyIdentifier": null,
-	"otherContribution": null,
-	"relatedPublication": null,
-	"releaseDate": "2006-08-01",
-	"repository": null,
-	"supportChannel": null,
-	"shortName": "MNI Colin27 Average Brain",
-	"usedSpecimen": null,
-	"versionIdentifier": "2008",
-	"versionInnovation": null
+  "axesOrigin": null,
+  "copyright": null,
+  "custodian": null,
+  "defaultImage": null,
+  "description": "The 'MNI Colin 27 Average Brain Stereotaxic Registration Model' is a stereotaxic average of T1-, T2- and PD-weighted MRI scans of the same individual (27, 12 and 12 scans, respectively).It was created at the Montreal Neurological Institute (MNI). The MRI scans were automatically registered to a common stereotaxic space using MRITOTAL (an automated volumetric registration procedure), resampled and intensity averaged.",
+  "digitalIdentifier": null,
+  "fullDocumentation": {
+    "@id": "https://www.mcgill.ca/bic/software/tools-data-analysis/anatomical-mri/atlases/colin-27-2008"
+  },
+  "fullName": "MNI Colin27 Average Brain Stereotaxic Registration Model",
+  "funding": null,
+  "homepage": "https://www.mcgill.ca/bic/software/tools-data-analysis/anatomical-mri/atlases",
+  "howToCite": "Please make sure to follow the copyright of the [MNI Colin27 Average Brain Stereotaxic Registration Model (high-resolution version 2008)](https://www.mcgill.ca/bic/software/tools-data-analysis/anatomical-mri/atlases/colin-27-2008). You may also cite its original publication, [Holmes et al. (1998)](http://doi.org/10.1097/00004728-199803000-00032), and the corresponding publication, [Aubert-Broche et al. (2006)](https://doi.org/10.1016/j.neuroimage.2006.03.052).",
+  "isAlternativeVersionOf": [
+    {
+      "@id": "https://openminds.om-i.org/instances/commonCoordinateSpaceVersion/MNI-Colin27_1998"
+    }
+  ],
+  "isNewVersionOf": null,
+  "keyword": null,
+  "license": null,
+  "nativeUnit": {
+    "@id": "https://openminds.om-i.org/instances/unitOfMeasurement/millimeter"
+  },
+  "ontologyIdentifier": null,
+  "otherContribution": null,
+  "relatedPublication": [
+    {
+      "@id": "https://doi.org/10.1097/00004728-199803000-00032"	  
+    },
+    {
+      "@id": "https://doi.org/10.1016/j.neuroimage.2006.03.052"	  
+    }
+  ],
+  "releaseDate": "2006-08-01",
+  "repository": null,
+  "shortName": "MNI Colin27 Average Brain",
+  "supportChannel": null,
+  "usedSpecimen": null,
+  "versionIdentifier": "2008",
+  "versionInnovation": "This is the high-resolution version of the common coordinate space. These volumes are defined at a 0.5 mm isotropic voxel grid in Talairach space, with dimensions 362x434x362 (XxYxZ) and start coordinates −90,−126,−72 (x,y,z)."
```

Comment:
is dimension and start coordinates different between the versions?

Reply:
It's the summary provided by the developers. It was the least time-consuming solution and reflects what the developers want to highlight about this version. I added the sources, which hopefully make this more user-friendly.